js/view.js

```diff
@@ -0,0 +1,21 @@
+var projectView = {};
+
+projectView.handleNav = function() {
+  $('.main-nav').on('click', '.tab', function() {
+    $('.content-to-toggle').hide();
+    $('#' + $(this).data('content')).fadeIn();
+  });
+  $('.main-nav .tab:first').click();
+};
+
+projectView.handleTeasers = function() {
+  $('.projects-body *:nth-of-type(n+2)').hide();
+  $('#projects').on('click', 'a.read-on', function(e) {
+    e.preventDefault();
+    $(this).parent().find('*').show();
+    $(this).hide();
+  });
```

Comment:
Great this is working for the read on button.
One way you could do the show less:

- you would set it up to change the text with the click. something like `$('this').text('Show less')`

- at the same time change the class from read-on to `.show-less`.

- Then set up another event handler doing the opposite by searching for `.show-less` on a click.